tests/api.suite.dist.yml

```diff
@@ -0,0 +1,12 @@
+actor: ApiTester
+modules:
+    enabled:
+        - \App\Tests\Helper\Api
+        - REST:
+            url: http://127.0.0.1/
```

Comment:
Why 127.0.0.1 ?

Reply:
That's local php server url in CircleCi build